tests/unit/test_certificates_integration.py

```diff
@@ -12,24 +12,28 @@
 
 import tls_relation
 
+from .conftest import GATEWAY_CLASS_CONFIG, TEST_EXTERNAL_HOSTNAME_CONFIG
+
 
-@pytest.mark.usefixtures("patch_load_incluster_config")
 def test_generate_password(harness: Harness):
     """
     arrange: given a charm with no connectable container.
     act: when agent relation joined event is fired.
     assert: the event is deferred.
```

Comment:
I'm wondering if the docstrings for this tests should be updated :)

Reply:
yep, I updated the docstring!